src/setViewport.js

```diff
@@ -35,12 +35,12 @@ export default function (element, viewport) {
   // Prevent window width from being too small (note that values close to zero are valid and can occur with
   // PET images in particular)
   if (enabledElement.viewport.voi.windowWidth) {
-    enabledElement.viewport.voi.windowWidth = Math.max(viewport.voi.windowWidth, MIN_WINDOW_WIDTH);
+    enabledElement.viewport.voi.windowWidth = Math.max(enabledElement.viewport.voi.windowWidth, MIN_WINDOW_WIDTH);
```

Comment:
`viewport.voi.windowWidth` and `enabledElement.viewport.voi.windowWidth` should be equal given the merge on lines 26-32.